proprietary/purmerend-design-tokens/figma/figma.tokens.json

```diff
@@ -1495,7 +1498,9 @@
           "fill-1": { "$value": "{purmerend.color.white-1}" },
           "fill-2": { "$value": "{purmerend.color.white-1}" },
           "text-1": { "$value": "{purmerend.color.white-1}" },
-          "text-2": { "$value": "{purmerend.color.white-1}" }
+          "text-2": { "$value": "{purmerend.color.white-1}" },
+          "text-3": { "$value": "{purmerend.color.blue-44}" },
+          "text-4": { "$value": "{purmerend.color.blue-44}" }
```

Comment:
Zelfde als bij eerdere wijziging: had dit niet gewoon in text-2 ingevuld kunnen worden?

En klopt de kleur uberhaupt wel in deze scale? Is blue-44 goed leesbaar ten opzichte van bg-1?

Reply:
Link text focus color, moet elke component waar ik aan werk alles in dat component uit hetzelfde scale komen?